CRITICAL - 2023-03-21 09:00:12 --> Undefined index: tahun
in APPPATH\Views\v_dashboard_siswa.php on line 48.
 1 APPPATH\Views\v_dashboard_siswa.php(48): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: tahun', 'APPPATH\\Views\\v_dashboard_siswa.php', 48, [...])
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_dashboard_siswa.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('v_dashboard_siswa', [], true)
 5 APPPATH\Views\tampilan.php(413): view('v_dashboard_siswa')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tampilan.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('tampilan', [], true)
 9 APPPATH\Controllers\Loginsiswa.php(33): view('tampilan', [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\loginsiswa->index()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\loginsiswa))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:17:19 --> mysqli_sql_exception: Unknown column 'tbl_jadwal.nis' in 'on clause' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelKrs.php(54): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\siakad2\app\Controllers\Krs.php(34): App\Models\ModelKrs->daftarap('25')
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Krs->index()
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Krs))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:17:19 --> Unknown column 'tbl_jadwal.nis' in 'on clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_jadwal`
LEFT JOIN `tbl_guru` ON `tbl_guru`.`nip` = `tbl_jadwal`.`nip`
LEFT JOIN `tbl_mapel` ON `tbl_mapel`.`id_mapel` = `tbl_jadwal`.`id_mapel`
LEFT JOIN `tbl_kelas` ON `tbl_kelas`.`id_kelas` = `tbl_jadwal`.`id_kelas`
LEFT JOIN `tbl_siswa` ON `tbl_siswa`.`nis` = `tbl_jadwal`.`nis`
WHERE `tbl_jadwal`.`id_kelas` = :tbl_jadwal.id_kelas:', [...], false)
 2 APPPATH\Models\ModelKrs.php(54): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Krs.php(34): App\Models\ModelKrs->daftarap('25')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Krs->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Krs))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 09:21:49 --> Undefined index: id_siswa
in APPPATH\Controllers\Krs.php on line 35.
 1 APPPATH\Controllers\Krs.php(35): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: id_siswa', 'APPPATH\\Controllers\\Krs.php', 35, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Krs->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Krs))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:37:30 --> mysqli_sql_exception: Duplicate entry 'admin' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\User.php(76): App\Models\ModelUser->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\User->add()
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:37:30 --> Duplicate entry 'admin' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`nama_user`, `username`, `password`, `foto`) VALUES (:nama_user:, :username:, :password:, :foto:)', [...], false)
 2 APPPATH\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\User.php(76): App\Models\ModelUser->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->add()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:37:56 --> mysqli_sql_exception: Duplicate entry 'admin' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\User.php(76): App\Models\ModelUser->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\User->add()
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:37:56 --> Duplicate entry 'admin' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`nama_user`, `username`, `password`, `foto`) VALUES (:nama_user:, :username:, :password:, :foto:)', [...], false)
 2 APPPATH\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\User.php(76): App\Models\ModelUser->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->add()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:38:37 --> mysqli_sql_exception: Duplicate entry 'admin' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\User.php(76): App\Models\ModelUser->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\User->add()
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:38:37 --> Duplicate entry 'admin' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`nama_user`, `username`, `password`, `foto`) VALUES (:nama_user:, :username:, :password:, :foto:)', [...], false)
 2 APPPATH\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\User.php(76): App\Models\ModelUser->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->add()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:39:06 --> mysqli_sql_exception: Duplicate entry 'admin' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\User.php(76): App\Models\ModelUser->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\User->add()
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:39:06 --> Duplicate entry 'admin' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`nama_user`, `username`, `password`, `foto`) VALUES (:nama_user:, :username:, :password:, :foto:)', [...], false)
 2 APPPATH\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\User.php(76): App\Models\ModelUser->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->add()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:40:56 --> mysqli_sql_exception: Duplicate entry 'admin' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\User.php(76): App\Models\ModelUser->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\User->add()
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:40:56 --> Duplicate entry 'admin' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`nama_user`, `username`, `password`, `foto`) VALUES (:nama_user:, :username:, :password:, :foto:)', [...], false)
 2 APPPATH\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\User.php(76): App\Models\ModelUser->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->add()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:41:27 --> mysqli_sql_exception: Duplicate entry 'admin' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\User.php(76): App\Models\ModelUser->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\User->add()
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:41:27 --> Duplicate entry 'admin' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`nama_user`, `username`, `password`, `foto`) VALUES (:nama_user:, :username:, :password:, :foto:)', [...], false)
 2 APPPATH\Models\ModelUser.php(23): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\User.php(76): App\Models\ModelUser->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->add()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 09:57:18 --> mysqli_sql_exception: Unknown column 'smt' in 'order clause' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelMapel.php(23): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\siakad2\app\Controllers\Mapel.php(41): App\Models\ModelMapel->allDataMapel('2')
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Mapel->detail('2')
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 09:57:18 --> Unknown column 'smt' in 'order clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_mapel`
WHERE `id_kelas` = :id_kelas:
ORDER BY `smt` DESC', [...], false)
 2 APPPATH\Models\ModelMapel.php(23): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Mapel.php(41): App\Models\ModelMapel->allDataMapel('2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->detail('2')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 10:01:21 --> mysqli_sql_exception: Duplicate entry 'BHS' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelMapel.php(28): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\Mapel.php(87): App\Models\ModelMapel->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 10:01:21 --> Duplicate entry 'BHS' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_mapel` (`kode_mapel`, `mapel`, `id_kelas`) VALUES (:kode_mapel:, :mapel:, :id_kelas:)', [...], false)
 2 APPPATH\Models\ModelMapel.php(28): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Mapel.php(87): App\Models\ModelMapel->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 10:02:28 --> mysqli_sql_exception: Duplicate entry 'BHS' for key 'PRIMARY' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelMapel.php(28): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\siakad2\app\Controllers\Mapel.php(71): App\Models\ModelMapel->add(Array)
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 10:02:28 --> Duplicate entry 'BHS' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_mapel` (`kode_mapel`, `mapel`, `id_kelas`) VALUES (:kode_mapel:, :mapel:, :id_kelas:)', [...], false)
 2 APPPATH\Models\ModelMapel.php(28): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Mapel.php(71): App\Models\ModelMapel->add([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 10:06:07 --> strpos(): Offset not contained in string
in SYSTEMPATH\Validation\Validation.php on line 812.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'strpos(): Offset not contained in string', 'SYSTEMPATH\\Validation\\Validation.php', 812, [...])
 2 SYSTEMPATH\Validation\Validation.php(812): strpos('required|is_unique[tbl_mapel.kode_mapel,id,{id},id_kelas,{id_kelas}', '|', 68)
 3 SYSTEMPATH\Validation\Validation.php(140): CodeIgniter\Validation\Validation->splitRules([...])
 4 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 5 APPPATH\Controllers\Mapel.php(50): CodeIgniter\Controller->validate([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 10:06:16 --> strpos(): Offset not contained in string
in SYSTEMPATH\Validation\Validation.php on line 812.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'strpos(): Offset not contained in string', 'SYSTEMPATH\\Validation\\Validation.php', 812, [...])
 2 SYSTEMPATH\Validation\Validation.php(812): strpos('required|is_unique[tbl_mapel.kode_mapel,id,{id},id_kelas,{id_kelas}', '|', 68)
 3 SYSTEMPATH\Validation\Validation.php(140): CodeIgniter\Validation\Validation->splitRules([...])
 4 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 5 APPPATH\Controllers\Mapel.php(50): CodeIgniter\Controller->validate([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 10:08:27 --> strpos(): Offset not contained in string
in SYSTEMPATH\Validation\Validation.php on line 812.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'strpos(): Offset not contained in string', 'SYSTEMPATH\\Validation\\Validation.php', 812, [...])
 2 SYSTEMPATH\Validation\Validation.php(812): strpos('required|is_unique[tbl_mapel.kode_mapel,id_kelas,{id_kelas}', '|', 60)
 3 SYSTEMPATH\Validation\Validation.php(140): CodeIgniter\Validation\Validation->splitRules([...])
 4 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 5 APPPATH\Controllers\Mapel.php(50): CodeIgniter\Controller->validate([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 10:12:47 --> mysqli_sql_exception: Unknown column 'tbl_jadwal.kode_mapel' in 'on clause' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelJadwal.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\siakad2\app\Controllers\JadwalTk.php(45): App\Models\ModelJadwal->allData('3')
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\JadwalTk->detail_jadwal('3')
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\JadwalTk))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 10:12:47 --> Unknown column 'tbl_jadwal.kode_mapel' in 'on clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_jadwal`
LEFT JOIN `tbl_mapel` ON `tbl_mapel`.`kode_mapel` = `tbl_jadwal`.`kode_mapel`
LEFT JOIN `tbl_kelas` ON `tbl_kelas`.`id_kelas` = `tbl_jadwal`.`id_kelas`
LEFT JOIN `tbl_ta` ON `tbl_ta`.`id_ta` = `tbl_kelas`.`id_ta`
LEFT JOIN `tbl_guru` ON `tbl_guru`.`nip` = `tbl_jadwal`.`nip`
WHERE `tbl_jadwal`.`id_kelas` = :tbl_jadwal.id_kelas:
ORDER BY `tbl_mapel`.`smt` ASC', [...], false)
 2 APPPATH\Models\ModelJadwal.php(18): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\JadwalTk.php(45): App\Models\ModelJadwal->allData('3')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\JadwalTk->detail_jadwal('3')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\JadwalTk))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 10:13:09 --> mysqli_sql_exception: Unknown column 'tbl_jadwal.kode_mapel' in 'on clause' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelJadwal.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\siakad2\app\Controllers\JadwalTk.php(45): App\Models\ModelJadwal->allData('1')
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\JadwalTk->detail_jadwal('1')
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\JadwalTk))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 10:13:09 --> Unknown column 'tbl_jadwal.kode_mapel' in 'on clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_jadwal`
LEFT JOIN `tbl_mapel` ON `tbl_mapel`.`kode_mapel` = `tbl_jadwal`.`kode_mapel`
LEFT JOIN `tbl_kelas` ON `tbl_kelas`.`id_kelas` = `tbl_jadwal`.`id_kelas`
LEFT JOIN `tbl_ta` ON `tbl_ta`.`id_ta` = `tbl_kelas`.`id_ta`
LEFT JOIN `tbl_guru` ON `tbl_guru`.`nip` = `tbl_jadwal`.`nip`
WHERE `tbl_jadwal`.`id_kelas` = :tbl_jadwal.id_kelas:
ORDER BY `tbl_mapel`.`smt` ASC', [...], false)
 2 APPPATH\Models\ModelJadwal.php(18): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\JadwalTk.php(45): App\Models\ModelJadwal->allData('1')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\JadwalTk->detail_jadwal('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\JadwalTk))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 10:19:08 --> mysqli_sql_exception: Unknown column 'tbl_mapel.id_mapel' in 'on clause' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelJadwal.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\siakad2\app\Controllers\JadwalTk.php(45): App\Models\ModelJadwal->allData('1')
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\JadwalTk->detail_jadwal('1')
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\JadwalTk))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 10:19:08 --> Unknown column 'tbl_mapel.id_mapel' in 'on clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_jadwal`
LEFT JOIN `tbl_mapel` ON `tbl_mapel`.`id_mapel` = `tbl_jadwal`.`id_mapel`
LEFT JOIN `tbl_kelas` ON `tbl_kelas`.`id_kelas` = `tbl_jadwal`.`id_kelas`
LEFT JOIN `tbl_ta` ON `tbl_ta`.`id_ta` = `tbl_kelas`.`id_ta`
LEFT JOIN `tbl_guru` ON `tbl_guru`.`nip` = `tbl_jadwal`.`nip`
WHERE `tbl_jadwal`.`id_kelas` = :tbl_jadwal.id_kelas:
ORDER BY `tbl_mapel`.`smt` ASC', [...], false)
 2 APPPATH\Models\ModelJadwal.php(18): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\JadwalTk.php(45): App\Models\ModelJadwal->allData('1')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\JadwalTk->detail_jadwal('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\JadwalTk))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 21:07:14 --> mysqli_sql_exception: Unknown column '{id}' in 'where clause' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Validation\Rules.php(156): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Validation\Validation.php(312): CodeIgniter\Validation\Rules->is_unique('Agama dan Moral', 'kode_mapel', Array, NULL, 'mapel')
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('mapel', 'Aspek Perkemban...', 'Agama dan Moral', Array, Array)
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\siakad2\app\Controllers\Mapel.php(50): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Mapel->add('25')
#10 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
#11 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-21 21:07:14 --> Unknown column '{id}' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tbl_mapel`
WHERE `kode_mapel` = :kode_mapel:
AND `{id}` != :{id}:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\Rules.php(156): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\Rules->is_unique('Agama dan Moral', 'kode_mapel', [...], null, 'mapel')
 4 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('mapel', 'Aspek Perkembangan', 'Agama dan Moral', [...], [...])
 5 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Mapel.php(50): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('25')
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-21 21:26:41 --> mysqli_sql_exception: Unknown column 'smt' in 'order clause' in C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\siakad2\app\Models\ModelMapel.php(23): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\siakad2\app\Controllers\Mapel.php(41): App\Models\ModelMapel->allDataMapel('25')
#6 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Mapel->detail('25')
#7 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
#8 C:\xampp\htdocs\siakad2\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\siakad2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-21 21:26:41 --> Unknown column 'smt' in 'order clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_mapel`
WHERE `id_kelas` = :id_kelas:
ORDER BY `smt` DESC', [...], false)
 2 APPPATH\Models\ModelMapel.php(23): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Mapel.php(41): App\Models\ModelMapel->allDataMapel('25')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->detail('25')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:06:17 --> Undefined variable: kode_mapel
in APPPATH\Controllers\Mapel.php on line 91.
 1 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: kode_mapel', 'APPPATH\\Controllers\\Mapel.php', 91, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:06:58 --> Undefined variable: kode_mapel
in APPPATH\Controllers\Mapel.php on line 91.
 1 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: kode_mapel', 'APPPATH\\Controllers\\Mapel.php', 91, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:08:05 --> Undefined variable: kode_mapel
in APPPATH\Controllers\Mapel.php on line 91.
 1 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: kode_mapel', 'APPPATH\\Controllers\\Mapel.php', 91, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:10:23 --> Undefined variable: kode_mapel
in APPPATH\Controllers\Mapel.php on line 91.
 1 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: kode_mapel', 'APPPATH\\Controllers\\Mapel.php', 91, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:10:52 --> Undefined variable: id_mapel
in APPPATH\Controllers\Mapel.php on line 91.
 1 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: id_mapel', 'APPPATH\\Controllers\\Mapel.php', 91, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:12:06 --> Undefined variable: kode_mapel
in APPPATH\Controllers\Mapel.php on line 91.
 1 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: kode_mapel', 'APPPATH\\Controllers\\Mapel.php', 91, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:15:20 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(642): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(832): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Mapel.php(93): CodeIgniter\Model->__call('where', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:16:32 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(642): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(832): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Mapel.php(93): CodeIgniter\Model->__call('where', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:17:06 --> Undefined variable: kode_mapel
in APPPATH\Controllers\Mapel.php on line 93.
 1 APPPATH\Controllers\Mapel.php(93): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: kode_mapel', 'APPPATH\\Controllers\\Mapel.php', 93, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:18:37 --> Undefined variable: kode_mapel
in APPPATH\Controllers\Mapel.php on line 91.
 1 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: kode_mapel', 'APPPATH\\Controllers\\Mapel.php', 91, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:20:07 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(642): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(832): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Mapel.php(91): CodeIgniter\Model->__call('where', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->add('2')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-21 22:24:51 --> Invalid argument supplied for foreach()
in APPPATH\Views\master\mapel\v_detail_mapel.php on line 58.
 1 APPPATH\Views\master\mapel\v_detail_mapel.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argument supplied for foreach()', 'APPPATH\\Views\\master\\mapel\\v_detail_mapel.php', 58, [...])
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\master\\mapel\\v_detail_mapel.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('master/mapel/v_detail_mapel', [], true)
 5 APPPATH\Views\tampilan.php(413): view('master/mapel/v_detail_mapel')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\tampilan.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('tampilan', [], true)
 9 APPPATH\Controllers\Mapel.php(43): view('tampilan', [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Mapel->detail('2')
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mapel))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
